schemas/adaptive-card.json

```diff
@@ -866,6 +1110,26 @@
 				"id"
 			]
 		},
+		"InlineAction": {
+			"additionalProperties": true,
+			"type": "object",
+			"version": "1.2",
+			"description": "The inline action for the input. Typically displayed to the right of the input. It is strongly recommended that an icon on the action (which will be displayed instead of the title of the action).",
```

Comment:
>It is strongly recommended that an icon on the action (which will be displayed instead of the title of the action). [](start = 99, length = 115)

Not a sentence :P How about

"It is strongly recommended to provide an icon on the action (which will be displayed exclusively instead of the title of the action).

Reply:
odd... thought this was a straight copy/paste from the spec. I probably started editing it and then got sidetracked :D

---
In reply to: [289951728](https://github.com/microsoft/AdaptiveCards/pull/2998#discussion_r289951728) [](ancestors = 289951728)